model_compression_toolkit/exporter/fully_quantized/keras/builder/quantize_config_to_node.py

```diff
@@ -29,6 +29,12 @@
 from model_compression_toolkit.exporter.fully_quantized.keras.quantize_configs.weights_quantize_config import \
     WeightsQuantizeConfig
 
+SUPPORTED_QUANTIZATION_CONFIG = [WeightsQuantizeConfig,
```

Comment:
It can not be completely separated since the model exporter expects to receive an exportable model, and part of it is the configurations it uses for wrapping the layers (so these configurations are valid). Since this is the file that generates the configurations in the model_wrapper I think it can fit here